code/modules/stock_market/stocks.dm

```diff
@@ -146,7 +146,8 @@
 		current_value += i_hate_this_code
 	if (current_value < 5)
 		current_value = 5
-
+	else if (current_value > 1000)
```

Comment:
Is there also a cap to prevent it going beyond -1000 ?

Reply:
I've never seen it go negative although i've seen the company die.